packages/backend/src/config/config.production.ts

```diff
@@ -176,6 +176,9 @@ export function getProductionConfig(env: Env): Config {
         {
           chainId: ChainId.ETHEREUM,
           rpcUrl: env.string('DISCOVERY_ETHEREUM_RPC_URL'),
+          rpcGetLogsMaxRange: env.optionalInteger(
+            'DISCOVERY_ETHEREUM_RPC_GETLOGS_MAX_RANGE',
```

Comment:
We should add this env var to the read in this the section under `#-----DISCOVERY-----`